### PR TITLE
app submission tracker reindexer

### PR DESCRIPTION
**Pull request by snopoke**
@benrudolph

### CODE REVIEW

File: corehq/pillows/app_submission_tracker.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    iteration_key = "CouchAppFormSubmissionTrackerPillow_reindexer"
+    doc_provider = CouchDocumentProvider(iteration_key, doc_types=[
+        XFormInstance,
+        XFormArchived,
+        XFormError,
+        XFormDeprecated,
+        XFormDuplicate,
+        ('XFormInstance-Deleted', XFormInstance),
+        ('HQSubmission', XFormInstance),
+        SubmissionErrorLog,
```
Comment (snopoke) [inline corehq/pillows/app_submission_tracker.py]: Not sure if all these doc types are of interest

Comment (benrudolph) [inline corehq/pillows/app_submission_tracker.py]: couldn't hurt and i think most of them will be necessary. probably not the SubmssionErrorLogs though

File: corehq/pillows/app_submission_tracker.py [hunk 2]
```diff
@@ -94,7 +94,6 @@ def get_couch_app_form_submission_tracker_reindexer():
         XFormError,
         XFormDeprecated,
         XFormDuplicate,
-        ('XFormInstance-Deleted', XFormInstance),
```
Comment (benrudolph) [inline corehq/pillows/app_submission_tracker.py]: 👍

**Comment by benrudolph** [comment]
👍 on ✔️ . kicked build

**Pull request merged**